build/teamcity-roachtest-stress.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+generate_ssh_key
+log_into_gcloud
+export ROACHPROD_USER=teamcity
+
+build/builder/mkrelease.sh amd64-linux-gnu build bin/workload bin/roachtest bin/roachprod 
+
+build/teamcity-roachtest-invoke.sh \
+  --debug=true \
```

Comment:
It would be good to have this configurable (and default to false). It isn't always useful to leave the clusters around.

Reply:
True, done.